docs/revsets.md

```diff
@@ -88,19 +88,23 @@ revsets (expressions) as arguments.
 * `none()`: No commits. This function is rarely useful; it is provided for
   completeness.
 * `branches([pattern])`: All local branch targets. If `pattern` is specified,
-  branches whose name contains the given string are selected. For example,
-  `branches(push)` would match the branches `push-123` and `repushed` but not
-  the branch `main`. If a branch is in a conflicted state, all its possible
-  targets are included.
-* `remote_branches([branch_pattern[, [remote=]remote_pattern]])`: All remote
-  branch targets across all remotes. If just the `branch_pattern` is specified,
-  branches whose name contains the given string across all remotes are
-  selected. If both `branch_pattern` and `remote_pattern` are specified, the
-  selection is further restricted to just the remotes whose name contains
-  `remote_pattern`. For example, `remote_branches(push, ri)` would match the
-  branches `push-123@origin` and `repushed@private` but not `push-123@upstream`
-  or `main@origin` or `main@upstream`. If a branch is in a conflicted state,
-  all its possible targets are included.
+  this selects the branches whose name match the given [string
+  pattern](#string-patterns). For example, `branches(push)` would match the
+  branches `push-123` and `repushed` but not the branch `main`. If a branch is
+  in a conflicted state, all its possible targets are included.
+
+*   `remote_branches([branch_pattern[, [remote=]remote_pattern]])`: All remote
```

Comment:
nit: unusual indent

Reply:
It's necessary for multiple paragraphs inside a list to work inside MkDocs. I will hopefully eventually figure out a more consistent way.